app/controllers/buyers/applications_controller.rb

```diff
@@ -24,19 +22,17 @@ def index
     activate_menu :audience, :applications, :listing
 
     @states = Cinstance.allowed_states.collect(&:to_s).sort
-    accessible_services = current_account.accessible_services
-    @services = accessible_services.includes(:application_plans)
+    @services = accessible_services
     @search = ThreeScale::Search.new(params[:search] || params)
-    @application_plans = current_account.application_plans.stock
-    @stock_and_custom_application_plans = current_account.application_plans.size
+    @application_plans = accessible_plans
```

Comment:
```suggestion
    @application_plans = accessible_plans.stock
```

Reply:
I wonder why the `.stock` isn't inside the `accessible_plans` method in this controller 🤔

Reply:
I don't know what `.stock` is for but it has been moved to the view I think